.github/workflows/labeler.yml

```diff
@@ -1,6 +1,11 @@
 name: 'Pull Request Labeler'
+
 on:
-    - pull_request_target
+    pull_request_target:
```

Comment:
This is what is making the CI task fail, right?

Reply:
No, it still failed after that :disappointed:   I'm trying to figure out what's going on, currently my best guess is that the latest version of https://github.com/actions/labeler requires different configuration.

Reply:
OK, I think I have this figured out:
- Check is failing because v5 introduced [new format for paths](https://github.com/marketplace/actions/labeler?version=v5.0.0). Our version is currently set to `main`, which worked fine until two weeks ago when v5 released.
- Check is still failing when I make the version fixed in PR, because it is still running actions/labeler@main

@izaera Any objections to merge the chore commit that sets version fixed to v4?

Reply:
:shrug: I've never touched that part. If it works, push it :sweat_smile:

Reply:
I think there is no way for me to know, before merging. It'll be fine, I'm sure 

<img src="https://media2.giphy.com/media/32mC2kXYWCsg0/giphy.gif"/>

Reply:
Leaving a note for @bryceosterhaus here. Setting of version from `main` to `v4` fixes the labeler issue, but I'm not sure if it goes again some wider dependencies convention in this repo. If yes, then we can spend a bit of time updating configuration format to the latest version.